lib/jsdom/living/cssom/StyleSheetList.webidl

```diff
@@ -0,0 +1,6 @@
+[Exposed=Window]
```

Comment:
Can you add a spec link? https://drafts.csswg.org/cssom/#the-stylesheetlist-interface